app/controllers/ConfirmationController.scala

```diff
@@ -60,6 +60,19 @@ class ConfirmationController @Inject()(appConfig: FrontendAppConfig,
       }).valueOr(failPage => failPage)
   }
 
+  def onPageRefresh(reference: String) = getData.async {
```

Comment:
I ve just moved the method at the top of the file to comapre it with `onPageLoad` - they quite similar - we might be able to merge the 2 at some point